Frameworks/include/ABSourceInternal.h

```diff
@@ -0,0 +1,28 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#pragma once
+
+#import "UWP/WindowsApplicationModelContacts.h"
+#import "ABRecordInternal.h"
+#import <AddressBook/ABRecord.h>
+
+@interface _ABSource : _ABRecord
+
+@property ABRecordID recordID;
+- (id)initWithRecordID:(ABRecordID)recordID;
```

Comment:
I meant the files themselves as well, since they're both new files.

---
In reply to: [74823842](https://github.com/Microsoft/WinObjC/pull/881#discussion_r74823842) [](ancestors = 74823842,74822609,74813579,74811477)